vit-servicing-station-tests/src/common/startup/server.rs

```diff
@@ -0,0 +1,71 @@
+use crate::common::startup::get_exe;
+use std::net::SocketAddr;
+use std::process::Command;
+use std::process::Stdio;
+use thiserror::Error;
+use vit_servicing_station_lib::server::settings::ServiceSettings;
+
+use crate::common::{paths::BLOCK0_BIN, server::Server, startup::get_available_port};
+
+pub struct Starter {
```

Comment:
Nitpicking: Can we use `ServerStarter`, `ServerBootstraper`, or something more specific here?